src/components/timeline/Lane.vue

```diff
@@ -63,12 +64,20 @@ export default {
     };
   },
   methods: {
+    onMouseOver() {
+      console.log("onMouseOver " + this.assignee.id);
```

Comment:
`console.log` calls should not be committed. Please remove them.

Reply:
Das Problem ist das die Task zwischen zwei Lanes abgelegt werden kann und es immernoch nicht dem neuen assignee zugeordnet wird.

![grafik](https://user-images.githubusercontent.com/65345146/87662818-19fe3000-c763-11ea-8fb9-3bc1fa7908ca.png)